packages/prepare/src/BasePrepare.ts

```diff
@@ -1,10 +1,27 @@
-import { Texture, BaseTexture } from '@pixi/core';
+import { Texture, BaseTexture, AbstractRenderer, Renderer } from '@pixi/core';
 import { Ticker, UPDATE_PRIORITY } from '@pixi/ticker';
 import { settings } from '@pixi/settings';
-import { Container } from '@pixi/display';
+import { Container, DisplayObject } from '@pixi/display';
 import { Text, TextStyle, TextMetrics } from '@pixi/text';
 import { CountLimiter } from './CountLimiter';
+import { CanvasPrepare } from 'pixi.js-legacy';
```

Comment:
Please don't add CanvasPrepare as a dependency, this is an inappropriate coupling between Prepare and CanvasPrepare. I know it's referenced in the types, but BasePrepare might be fine in its place.

Reply:
it seems it's added automatically by IDE. Should CanvasPrepare be replaced with BasePrepare or should it be completely removed?